Concentration/ViewController.swift

```diff
@@ -11,58 +11,54 @@ final class ViewController: UIViewController {
 
     // MARK: - Properties -
 
-    private static var emojiChoices = ["🦇", "😱", "🙀", "😈", "🎃", "👻", "🍭", "🍬", "🍎"]
+    private lazy var game = Concentration(numberOfCards: self.cardButtons.count, theme: .allCases.randomElement() ?? .animals)
```

Comment:
I assume we could use some better way of setting new game as we did in Matchismo however I was too lazy (pun intended) to do that

Reply:
and also you have getRandomTheme